examples/gameroom/README.md

```diff
@@ -31,9 +31,16 @@ below.
 1. To start Gameroom, run the following command from the Splinter root
    directory:
 
-     ```
-     docker-compose -f examples/gameroom/docker-compose.yaml up
-     ```
+    ```
+    docker-compose -f examples/gameroom/docker-compose.yaml up --build
+    ```
+
+    **Note:** To run Gameroom with experimental features enabled, set an
+    enviroment variable in your shell before running the command above. For
+    example: `export 'CARGO_ARGS=-- --features experimental'`. To go back to
+    building with default features, unset the evironment variable:
```

Comment:
environment is spelled wrong in both places: evironment  enviroment

Reply:
good catch! fixed now.